docker_launch.sh

```diff
@@ -1,21 +1,9 @@
-#!/bin/bash
+#!/bin/sh
 set -e
 
-SSH_KEY=${SSH_KEY:-"/root/.ssh/id_rsa"}
-
-if [ ! -f "$SSH_KEY" ]; then
-    mkdir -p /root/.ssh/
-    find /ssh/* ! -type l -print0 | xargs -0 cp -t /root/.ssh/
-fi
-chown root:root /root/.ssh/*
-
-eval `ssh-agent -s` && ssh-add $SSH_KEY
-
-if [ ! -f ./security.yml ]; then
-    ./security-setup --enable=false
-fi
-
+python2 -c "import docker_setup; docker_setup.link_or_generate_ssh_keys()"
```

Comment:
That sounds good. It could have different command line options for different phases, rather than being imported as well.